duqtools/create.py

```diff
@@ -2,14 +2,13 @@
 import shutil
 import stat
 from pathlib import Path
-from typing import Iterable, List
+from typing import Iterable
 
 import yaml
-from pydantic import DirectoryPath
 
-from duqtools.config import IDSOperation, WorkDirectory, cfg
+from duqtools.config import WorkDirectory, cfg
 
-from ._types import BaseModel
+from .cleanup import Runs
```

Comment:
I think this would be better as `from .config.cleanup import Runs` (see previous comment)